### DISCUSSION TITLE
Custom Checks - stored procedure

### DISCUSSION THREAD

**Discussion by BrubiSQL** [trimble-oss/dba-dash#223] [Q&A] [answered]
Hi,

In which database do I need to place stored procedure for Custom Checks?
Does it need to be in master database? Or I can place it in any other database?
I've tried placing it in one of my administrative databases on SQL server, but it's not working, when I place it in master it works. So, I'm not sure am I missing something or this is intended behaviour...

https://github.com/trimble-oss/dba-dash/blob/main/Docs/CustomChecks.md

**Comment by DavidWiseman** [comment, marked as answer]
It's the database that the DBA Dash agent connects to - to change this specify an initial catalog in the connection string for the source connection.
e.g. 
_Data Source=TheInstance;**Initial Catalog=YourUserDB;I**ntegrated Security=True;Encrypt=True;Trust Server Certificate=True;Application Name=DBADash_
Or using the GUI:
![image](https://user-images.githubusercontent.com/33157668/173019998-a5069ec1-c798-4060-9769-c60c6696eb9c.png)

Hope this helps

**Reply by BrubiSQL** [reply]
That explains a few things :)
Thank you for your help! :)

(I was making my own custom script to check for DAC port on named instance, and now it works \o/ )
